specs/lightclient/beacon-chain.md

```diff
@@ -211,52 +498,104 @@ def process_sync_committee(state: BeaconState, body: BeaconBlockBody) -> None:
 
 ```python
 def process_epoch(state: BeaconState) -> None:
-    process_justification_and_finalization(state)
-    process_rewards_and_penalties(state)
+    process_justification_and_finalization(state)  # [Updated in HF1]
+    process_rewards_and_penalties(state)  # [Updated in HF1]
     process_registry_updates(state)
     process_slashings(state)
     process_eth1_data_reset(state)
     process_effective_balance_updates(state)
     process_slashings_reset(state)
     process_randao_mixes_reset(state)
     process_historical_roots_update(state)
-    process_participation_record_updates(state)
-    # Light client patch
+    # [Added in HF1]
+    process_participation_flag_updates(state)
     process_sync_committee_updates(state)
+    # [Removed in HF1] -- process_participation_record_updates(state)
```

Comment:
Nitpicking: to make it more readable that `process_participation_flag_updates` replaced  `process_participation_record_updates`.
```suggestion
    # [Removed in HF1] -- process_participation_record_updates(state)
    # [Added in HF1]
    process_participation_flag_updates(state)
    process_sync_committee_updates(state)
```